### PR TITLE
Connection page

### PR DESCRIPTION
**Pull request by Marie-Garde**

### CODE REVIEW

**Comment by Thoms-C** [comment]
Tout est ok pour moi, cependant il faut juste changer em en rem pour le slogan et titleForm comme convenu

**Review by Thoms-C — DISMISSED** [review]

File: src/Components/Connexion.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  ContainerForm,
+  Logo,
+  ContainerLogo,
+  Slogan,
+  Span,
+  TitleForm,
+  H2,
+} from "../Styled-components/ConnexionStyle";
+
+export default function connexion() {
```
Comment (LoicBrassart) [inline src/Components/Connexion.jsx]: This is a React component: rename it to "Connection" (first capital letter + English name)

File: src/Components/Device.jsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const size = {
+  mobile: "825px",
+  desktop: "2560px",
+};
+
+export const device = {
```
Comment (LoicBrassart) [inline src/Components/Device.jsx]: Rename > "Device"

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Non file-specific issues:
- Rename "Assets" > "assets"
- Rename "Components" > "components"
- Rename "Styled-components" > "styled-components"
- Write some doc about why you created the "styled-components" and what's to be found in it

**Review by Thoms-C — APPROVED** [review]

**Review by Thoms-C — DISMISSED** [review]

**Review by Thoms-C — COMMENTED** [review]
You need to rename Src/components/Connexion to Src/components/Connection

**Review by Thoms-C — DISMISSED** [review]
Nice !

**Review by Thoms-C — DISMISSED** [review]
Seems ok

**Review by LoicBrassart — DISMISSED** [review]
No more Evils
(But beware, you should not call alll your commits "Prettify", that's a nonsense)

**Review by Thoms-C — APPROVED** [review]
Ok !

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**